### ISSUE TITLE
Infrastructure updates for updated deployment workflow

### ISSUE THREAD

**Issue by marjo-luc** [MAAP-Project/ZenHub#683]
This ticket captures the work needed to accommodate the updated deployment workflow. See discussion [here](https://app.zenhub.com/workspaces/maap-platform-5ee7f0d1f440de0024cd359b/issues/gh/maap-project/zenhub/308).

- [ ] Update branch protection (PR and merging rules) of maap-workspaces and any other applicable repos

- [ ] Update devfile branch references

- [ ] Create git actions to copy images built in the development venue over to the test and ops venues when PRs from the development venue are merged in the test and ops venues, respectively.

- [ ] Document updated deployment workflow

**Comment by grallewellyn** [comment]
Additional question for our discussion about develop branches: 
Say I publish a new version of a jupyter extension and the only update I make to the maap-workspaces dockerfile is changing the install version for said jupyter extension. I then create a PR for this new version change into the develop branch of maap-workspaces. Would it be expected for the reviewer of my PR for maap-workspaces to also review the changes in my jupyter extension git repo?

**Comment by marjo-luc** [comment]
No, unless the developer specifically requests it. That said, the developer may want to consider highlighting changes in the code (i.e. the jupyter lab extension) that may affect other codebases so the reviewer may be better informed when reviewing. 

The developer is expected to test the content of the PR. Once the PR is merged, it will go through QA for further testing, so having the reviewer perform testing as well is likely unnecessary.
